src/lib.rs

```diff
@@ -64,6 +64,8 @@ pub struct Metadata {
     pub authors: Cow<'static, str>,
     /// The URL of the crate's website
     pub homepage: Cow<'static, str>,
+    /// The support information
+    pub supports: Cow<'static, str>,
```

Comment:
Adding this field is a breaking change.  If we're already making a breaking change related to this, maybe we should change the API so we can further extend this without more breaking changes.

For example, we could switch `Metadata` to being made by builder methods.  We'd likely want to make each field `Option` and not require any field?